apps/web/package.json

```diff
@@ -7,6 +7,7 @@
     "build": "next build",
     "start": "next start",
     "lint": "eslint --fix './'",
+    "test": "jest --detectOpenHandles",
```

Comment:
I thought `--detectOpenHandles` is only needed when you `--forceExit`. Why do we keep using it _everywhere_?